classes/TranslatableBehavior.php

```diff
@@ -97,6 +97,20 @@ public function initTranslatableContext()
         $this->translatableDefault = $translate->getDefaultLocale();
     }
 
+    /**
+     * Add an attribute to the model's translatable array.
+     * @param  string  $attribute
```

Comment:
See https://github.com/wintercms/storm/blob/develop/src/Database/Traits/Purgeable.php#L43-L48 for what sort of input is generally accepted by the add*() methods, would like to support the same things here. Also would like to use real typehints instead of the docblock ones.